ray-operator/controllers/raycluster_controller.go

```diff
@@ -248,6 +248,35 @@ func (r *RayClusterReconciler) reconcilePods(instance *rayiov1alpha1.RayCluster)
 			}
 		}
 		diff := *worker.Replicas - int32(len(runningPods.Items))
+
+		//// SriramQ: How do I create a feature flag to guard the new functionality?
+		featureFlag := true
```

Comment:
This is a good question. We didn't have global feature gate mechanism at this moment. Let's use this temporarily.  

I create https://github.com/ray-project/kuberay/issues/211 for the feature gate implementation and discussion

Reply:
I would like to do better - we should be able to provide this as a startup option. I think I know how to do this - please wait for my next code update.